.circleci/src/jobs/deploy_administration.yml

```diff
@@ -0,0 +1,34 @@
+docker:
+  - image: cimg/base:2022.09
+parameters:
```

Comment:
I think it is better to pass a enum ['production' or 'beta'] an use this to evaluate which server and fingerprint should be used.

Reply:
hm i understand the point of readability but it requires way more code to implement the different conditions. not sure if thats an improvement in that case

Reply:
okay maybe then i  don't know how to proper implement this in circle ci
I thought i have to pass f.e. a boolean with `production=true`
Then i use a `when` condition and use there the particular values for production and the else condition for beta.
So if we take `deliver_administration` job as an example, i have to do the run steps `Upload package` and `install package` in both cases with different values and duplicate the lines of code